src/components/Page/Page.js

```diff
@@ -68,6 +76,21 @@ export default class Page extends Component {
     )
   }
 
+  findTagDocs (tagHandle, navigation) {
```

Comment:
With this approach, the UI component is getting 'smart', as it is now understanding the relationship between `services` and `navigation`.

Another approach is to keep the UI component really simple, and handle `externalDocs` at the parser level.

What do you reckon @brendo ?